probe/client/mongo/mongo.go

```diff
@@ -50,6 +52,8 @@ func New(opt conf.Options) Mongo {
 			opt.Host, opt.Timeout().Milliseconds())
 	}
 
+	log.Debugf("[%s / %s / %s] - Connection - %s", opt.ProbeKind, opt.ProbeName, opt.ProbeTag, conn)
```

Comment:
I think this should be fine because the config.yaml also contains the username & password.

and this is the debug log, by default, it is disabled. and to enable that, we need to configure the config.yaml, and the config.yaml exposes everything.